data-service/src/test/java/com/amazonaws/blox/dataservice/repository/EnvironmentRepositoryDDBTest.java

```diff
@@ -164,29 +161,28 @@ public void testListEnvironmentsWithEnvironmentNamePrefix() throws Exception {
     when(environmentMapper.toEnvironment(environmentDDBRecord)).thenReturn(environment);
 
     final List<Environment> result =
-        environmentRepositoryDDB.listEnvironments(cluster, Optional.of(environmentNamePrefix));
+        environmentRepositoryDDB.listEnvironments(cluster, environmentNamePrefix);
 
     verify(dynamoDBMapper)
         .query(eq(EnvironmentDDBRecord.class), ddbQueryExpressionCaptor.capture());
     verify(environmentMapper).toEnvironment(environmentDDBRecord);
 
     final EnvironmentDDBRecord queriedEnvironmentDDBRecord =
         (EnvironmentDDBRecord) ddbQueryExpressionCaptor.getValue().getHashKeyValues();
-    final Map queryConditions = ddbQueryExpressionCaptor.getValue().getRangeKeyConditions();
-
-    assertThat(
-        queriedEnvironmentDDBRecord.getAccountIdCluster(), is(cluster.generateAccountIdCluster()));
-    assertThat(queryConditions.entrySet().size(), is(1));
-    final Map.Entry conditionEntry = (Map.Entry) queryConditions.entrySet().iterator().next();
-    assertThat(conditionEntry.getKey(), is(ENVIRONMENT_NAME_RANGE_KEY));
-    assertThat(
-        ((Condition) conditionEntry.getValue()).getComparisonOperator(),
-        is(ComparisonOperator.BEGINS_WITH.toString()));
-    assertThat(
-        ((Condition) conditionEntry.getValue()).getAttributeValueList().get(0),
-        is(new AttributeValue().withS(environmentNamePrefix)));
-    assertThat(result.size(), is(1));
-    assertThat(result.get(0), is(environment));
+    final Map<String, Condition> queryConditions =
+        ddbQueryExpressionCaptor.getValue().getRangeKeyConditions();
+
+    assertThat(queriedEnvironmentDDBRecord.getAccountIdCluster())
+        .isEqualTo(cluster.generateAccountIdCluster());
+    assertThat(queryConditions).isNotEmpty().hasSize(1);
+    assertThat(queryConditions).containsKey(ENVIRONMENT_NAME_RANGE_KEY);
+    assertThat(queryConditions)
```

Comment:
Protip: For comparing maps, you can use `assertThat(queryConditions).contains(entry(...key..., ...value...)`